boogiestats/templates/boogie_ui/scorecard.html

```diff
@@ -0,0 +1,172 @@
+    }
+
+    const canvas = document.getElementById('statcard')
+    const context = canvas.getContext('2d')
+    const nodec = '{{ score.comment }}'.includes('No Dec/WO')
+    let diffColor
+    switch ('{{ score.song.chart_info.diff }}') {
+        case 'Beginner':
+            diffColor = '#a355b8'
+            break
+        case 'Easy':
+            diffColor = '#1ec51d'
+            break
+        case 'Medium':
+            diffColor = '#d6db41'
+            break
+        case 'Hard':
+            diffColor = '#ba3049'
+            break
+        case 'Challenge':
+            diffColor = '#2691c5'
+            break
+        default:
+            diffColor = '#ffffff'
+            break
+    }
+
+    barWidth = 306
+
+    const sum = parseInt('{{ score.total_steps }}')
+    const p = parseInt('{{ score.fantastics_plus }}') / sum * barWidth;
+    const f = parseInt('{{ score.fantastics }}') / sum * barWidth;
+    const e = parseInt('{{ score.excelents }}') / sum * barWidth;
+    const g = parseInt('{{ score.greats }}') / sum * barWidth;
+    const d = parseInt('{{ score.decents }}') / sum * barWidth;
+    const w = parseInt('{{ score.wayoffs }}') / sum * barWidth;
+    const m = parseInt('{{ score.misses }}') / sum * barWidth;
+
+    context.fillStyle = '#000000'
+    context.fillRect(0, 0, 400, 600)
+
+    context.fillStyle = diffColor
+    context.fillRect(47, 127, 65, 65)
+
+    context.textAlign = 'center'
+    if ('{{score.song.gs_ranked}}' === 'True') {
+        context.fillStyle = '#38abb5'
+        context.fillRect(350, 20, 30, 30)
+        context.fillStyle = '#ffffff'
+        context.font = '30px miso'
+        context.fillText('GS', 365, 45)
+
+    }
+
+    context.strokeStyle = '#ffffff'
+    context.rect(47, 210, 306, 30);
+    context.stroke()
+
+    let start = 47
+    context.fillStyle = '#21cce8'
+    context.fillRect(start, 210, p, 30)
+    start += p || 0
+    context.fillStyle = '#e0e0e0'
+    context.fillRect(start, 210, f, 30)
+    start += f || 0
+    context.fillStyle = '#e29c18'
+    context.fillRect(start, 210, e, 30)
+    start += e || 0
+    context.fillStyle = '#66c955'
+    context.fillRect(start, 210, g, 30)
+    start += g || 0
+    context.fillStyle = '#6365c9'
+    context.fillRect(start, 210, d, 30)
+    start += d || 0
+    context.fillStyle = '#e08c36'
+    context.fillRect(start, 210, w, 30)
+    start += w || 0
+    context.fillStyle = '#ff0000'
+    context.fillRect(start, 210, m, 30)
+
+    context.fillStyle = '#000000'
+    context.font = '80px Wendy'
+    context.fillText('{{ score.song.chart_info.diff_number }}' || '?', 86, 180)
+    context.font = '30px Wendy'
+    context.fillStyle = '#ffffff'
+    if('{{ score.song.chart_info.steps_type}}'.includes('single')) context.fillText('SINGLE', 81, 123)
+    if('{{ score.song.chart_info.steps_type}}'.includes('double')) context.fillText('DOUBLE', 81, 123)
+    
+    context.font = '50px miso'
+    context.fillText('BoogieStats', 200, 45)
```

Comment:
I think we could get rid of the big branding at the top, there's a smaller one in the footer that's also a full address which might be more useful for getting new people on